<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>950</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>167</red>
        <green>199</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>167</red>
        <green>199</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>167</red>
        <green>199</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>167</red>
        <green>199</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>50</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>soul_sync_logo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>250</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: #2C3E50;
}</string>
    </property>
    <property name="text">
     <string>Soul Sync</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="result_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>801</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: #2C3E50;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>471</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    background-color: #F5F7FA; 
    color: #333333;
    font-size: 16px;
    font-family: &quot;Arial&quot;, sans-serif;
    border: 2px solid #4682B4; 
    border-radius: 8px; 
    padding: 10px;
}


QGroupBox:focus {
    border: 2px solid #5A9BD6; 
    background-color: #FFFFFF; 
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="tips_label">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>56</y>
       <width>465</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{background-color: #D6E6F2;
 color: #6D6D6D;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{color: #2C3E50;
}</string>
     </property>
     <property name="text">
      <string>Stress Relief Tips </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>420</y>
      <width>491</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    background-color: #F5F7FA; 
    color: #333333;
    font-size: 16px;
    font-family: &quot;Arial&quot;, sans-serif;
    border: 2px solid #4682B4; 
    border-radius: 8px; 
    padding: 10px;
}


QGroupBox:focus {
    border: 2px solid #5A9BD6; 
    background-color: rgba(214, 230, 242, 0.85);
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{color: #2C3E50;
}</string>
     </property>
     <property name="text">
      <string>Exercises for Relaxation</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="exercise_label">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>56</y>
       <width>485</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{background-color: #D6E6F2;
 color: #6D6D6D;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>730</y>
      <width>1151</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    background-color: #F5F7FA; 
    color: #333333;
    font-size: 16px;
    font-family: &quot;Arial&quot;, sans-serif;
    border: 2px solid #4682B4; 
    border-radius: 8px; 
    padding: 10px;
}


QGroupBox:focus {
    border: 2px solid #5A9BD6; 
    background-color: #FFFFFF; 
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>110</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 1px solid #5a91c5;  
    border-radius: 8px;  
    background-color: #a7c7e7; 
    padding: 10px;  
    margin-top: 10px;  
    font: bold 12pt &quot;Arial&quot;; 
    color: #2c3e50;  
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="owner_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="quote_label">
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>70</y>
       <width>1111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{color: #2C3E50;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Subheading Semibold</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{color: #2C3E50;
}</string>
     </property>
     <property name="text">
      <string>Uplifting Quote</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="menu_btn">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #4682B4;
    color: white;
    font-size: 14px;
    font-weight: bold;
    border: none;
    border-radius: 12px; 
    padding: 10px 20px;
    min-width: 100px;
    transition: background-color 0.3s ease-in-out, transform 0.1s;
}


QPushButton:hover {
    background-color: #5A9BD6;
}


QPushButton:pressed {
    background-color: #36648B;
    transform: scale(0.98); 
}
</string>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
